eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java

```diff
@@ -55,6 +56,10 @@ public boolean loadFromCache(InitializationCallback callback) {
                     InputStreamReader reader = cacheFile.getInputReader();
                     RandomizationConfigResponse configResponse = gson.fromJson(reader, RandomizationConfigResponse.class);
                     reader.close();
+                    if (configResponse == null || configResponse.getFlags() == null) {
+                        // Invalid cached configuration, initialize as an empty map and delete file
+                        throw new JsonSyntaxException("Configuration file missing flags");
```

Comment:
Make a more helpful error message just in case this ever does happen for whatever reason